frontend/packages/vkt/src/components/clerkExamEvent/overview/ClerkExamEventDetailsFields.tsx

```diff
@@ -98,26 +103,40 @@ export const ClerkExamEventDetailsFields = ({
           className="rows gapped"
           data-testid="clerk-exam-event__basic-information__date"
         >
-          <H3>{t('header.date')}</H3>
+          <H3>{t('date')}</H3>
           <CustomDatePicker
             value={dayjs(examEvent?.date) ?? null}
             setValue={onDateChange('date')}
             disabled={editDisabled}
           />
         </div>
+      </div>
+      <div className="margin-top-lg grid-columns gapped">
         <div
           className="rows gapped"
           data-testid="clerk-exam-event__basic-information__registrationCloses"
         >
-          <H3>{t('header.registrationCloses')}</H3>
+          <H3>{t('registrationCloses')}</H3>
           <CustomDatePicker
             value={dayjs(examEvent?.registrationCloses) ?? null}
             setValue={onDateChange('registrationCloses')}
             disabled={editDisabled}
           />
```

Comment:
Tähän vois ehkä laittaa kans tuon `maxDate={examEvent?.date.subtract(1, 'd')}` niin käyttöliittymä vähän ohjais paremmin kun tuo tallennus tän tarkistaa eikä anna mitään virhettä jos väärin.

Reply:
Meinasin että oon laiska ja en lisäile tänne sen enempiä kun tuskin virkailijat pahemmin näitä tietoja tarvii muutella tilaisuuden luonnin jälkeen. Mut laitan ja ehkä hyvä pistääkin niin on sitten konsistentti tuon luonnin kanssa.